src/Tools/Source/RunTests/Program.cs

```diff
@@ -42,7 +44,9 @@ internal static int Main(string[] args)
                 cts.Cancel();
             };
 
-            return Run(options, cts.Token).GetAwaiter().GetResult();
+            var result = Run(options, cts.Token).GetAwaiter().GetResult();
+            CheckTotalDumpFilesSize();
```

Comment:
Here is the mental exercise that I do for changes like this:

1. We hit this limit at some point in the future.
1. We will curse and say "oh man, we had that bug, had the full dump and threw it away for no good reason".
1. We will send a PR to double the limit 

Whenever we have a limit like this and the response to violations is just increasing the limit we should really just not limit the dumps at all.

I won't block the PR on this but it's a restriction we should **not** be adding.